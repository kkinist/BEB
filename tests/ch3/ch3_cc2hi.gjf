%chk=ch3.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check

BEB step 9: high-spin CCSD(T) for dication of ch3

2 4

C H 0
cc-pVTZ
****

