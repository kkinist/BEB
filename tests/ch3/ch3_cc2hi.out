 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-28070.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     28072.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch3.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check
 ---------------------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1,116=-2/1,2,3;
 4/5=1/1;
 5/5=2,8=3,13=1,38=6/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 -------------------------------------------------
 BEB step 9: high-spin CCSD(T) for dication of ch3
 -------------------------------------------------
 Structure from the checkpoint file:  "ch3.chk"
 Charge =  2 Multiplicity = 4
 Redundant internal coordinates found in file.
 C,0,0.,-0.0000000011,0.
 H,0,0.000000002,-1.0827517802,0.
 H,0,-0.9376905476,0.5413758867,0.
 H,0,0.9376905457,0.5413758901,0.
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000   -1.082752    0.000000
      3          1           0       -0.937691    0.541376    0.000000
      4          1           0        0.937691    0.541376    0.000000
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4
     1  C    0.000000
     2  H    1.082752   0.000000
     3  H    1.082752   1.875381   0.000000
     4  H    1.082752   1.875381   1.875381   0.000000
 Stoichiometry    CH3(2+,4)
 Framework group  D3H[O(C),3C2(H)]
 Deg. of freedom     1
 Full point group                 D3H     NOp  12
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    1.082752    0.000000
      3          1           0       -0.937691   -0.541376    0.000000
      4          1           0        0.937691   -0.541376    0.000000
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    285.1563071    285.1563071    142.5781536
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.8236000000D+04  0.5419783203D-03
      0.1235000000D+04  0.4192873817D-02
      0.2808000000D+03  0.2152216205D-01
      0.7927000000D+02  0.8353432195D-01
      0.2559000000D+02  0.2395828457D+00
      0.8997000000D+01  0.4428528419D+00
      0.3319000000D+01  0.3517995618D+00
 S   6 1.00       0.000000000000
      0.2808000000D+03 -0.5949224937D-04
      0.7927000000D+02 -0.1148158310D-02
      0.2559000000D+02 -0.1001913745D-01
      0.8997000000D+01 -0.6121949230D-01
      0.3319000000D+01 -0.1732698541D+00
      0.3643000000D+00  0.1072915192D+01
 S   1 1.00       0.000000000000
      0.9059000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1285000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.1871000000D+02  0.3942638716D-01
      0.4133000000D+01  0.2440889849D+00
      0.1200000000D+01  0.8154920089D+00
 P   1 1.00       0.000000000000
      0.3827000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1209000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1097000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.3180000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.7610000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      4 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    35 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     8 symmetry adapted cartesian basis functions of A2  symmetry.
 There are    22 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    15 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    30 symmetry adapted basis functions of A1  symmetry.
 There are     8 symmetry adapted basis functions of A2  symmetry.
 There are    20 symmetry adapted basis functions of B1  symmetry.
 There are    14 symmetry adapted basis functions of B2  symmetry.
    72 basis functions,   103 primitive gaussians,    80 cartesian basis functions
     5 alpha electrons        2 beta electrons
       nuclear repulsion energy         9.6437168407 Hartrees.
 NAtoms=    4 NActive=    4 NUniq=    2 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    72 RedAO= T EigKep=  2.77D-03  NBF=    30     8    20    14
 NBsUse=    72 1.00D-06 EigRej= -1.00D+00 NBFU=    30     8    20    14
 Initial guess from the checkpoint file:  "ch3.chk"
 B after Tr=     0.000000    0.000000    0.000000
         Rot=    1.000000    0.000000    0.000000    0.000000 Ang=   0.00 deg.
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (?A) (?A) (?B) (?B) (?C)
       Virtual   (?A) (?B) (?B) (?B) (?B) (?C) (?A) (?A) (?B) (?B)
                 (?D) (?D) (?A) (?A) (?B) (?B) (?C) (?D) (?D) (?B)
                 (?B) (?B) (?B) (?A) (?C) (?A) (?B) (?B) (?B) (?B)
                 (?D) (?D) (?A) (?C) (?A) (?B) (?B) (?D) (?D) (?C)
                 (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?A) (?C)
                 (?B) (?B) (?D) (?D) (?C) (?B) (?B) (?A) (?B) (?B)
                 (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Beta  Orbitals:
       Occupied  (?A) (?A)
       Virtual   (?B) (?B) (?C) (?A) (?B) (?B) (?B) (?B) (?C) (?A)
                 (?A) (?B) (?B) (?D) (?D) (?A) (?A) (?B) (?B) (?C)
                 (?D) (?D) (?B) (?B) (?B) (?B) (?A) (?C) (?A) (?B)
                 (?B) (?B) (?B) (?D) (?D) (?A) (?C) (?A) (?B) (?B)
                 (?D) (?D) (?C) (?A) (?B) (?B) (?D) (?D) (?B) (?B)
                 (?A) (?A) (?C) (?B) (?B) (?D) (?D) (?C) (?B) (?B)
                 (?A) (?B) (?B) (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Initial guess <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.5000 <S**2>= 3.7500 S= 1.5000
 Keep R1 and R2 ints in memory in symmetry-blocked form, NReq=5456056.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(UHF) =  -38.2018276074     A.U. after    9 cycles
            NFock=  9  Conv=0.27D-08     -V/T= 1.9804
 <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 1.5000 <S**2>= 3.7530 S= 1.5007
 <L.S>= 0.000000000000E+00
 Annihilation of the first spin contaminant:
 S**2 before annihilation     3.7530,   after     3.7500
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    72
 NBasis=    72 NAE=     5 NBE=     2 NFC=     1 NFV=     0
 NROrb=     71 NOA=     4 NOB=     1 NVA=    67 NVB=    70

 **** Warning!!: The largest alpha MO coefficient is  0.11855371D+02


 **** Warning!!: The largest beta MO coefficient is  0.11849028D+02

 Semi-Direct transformation.
 ModeAB=           4 MOrb=             4 LenV=    1048251962
 LASXX=       174447 LTotXX=      174447 LenRXX=      358619
 LTotAB=      184172 MaxLAS=      685008 LenRXY=           0
 NonZer=      533066 LenScr=     1571840 LnRSAI=      685008
 LnScr1=     1571840 LExtra=           0 Total=      4187307
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=1 Pass  1:  I=   1 to   4.
 (rs|ai) integrals will be sorted in core.
 ModeAB=           4 MOrb=             1 LenV=    1048251962
 LASXX=        45880 LTotXX=       45880 LenRXX=       89121
 LTotAB=       43241 MaxLAS=      171252 LenRXY=           0
 NonZer=      135001 LenScr=      785920 LnRSAI=      171252
 LnScr1=      785920 LExtra=           0 Total=      1832213
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=2 Pass  1:  I=   1 to   1.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.5784175873D-02 E2=     -0.2158973292D-01
     alpha-beta  T2 =       0.2163017102D-01 E2=     -0.6615488822D-01
     beta-beta   T2 =       0.0000000000D+00 E2=      0.0000000000D+00
 ANorm=    0.1013614496D+01
 E2 =    -0.8774462115D-01 EUMP2 =    -0.38289572228592D+02
 (S**2,0)=  0.37530D+01           (S**2,1)=  0.37502D+01
 E(PUHF)=      -0.38202551834D+02        E(PMP2)=      -0.38289955205D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=5403492.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 E(PMP3)=      -0.38303815413D+02
 MP4(R+Q)=  0.19224119D-02
 E3=       -0.14077432D-01        EUMP3=      -0.38303649661D+02
 E4(DQ)=   -0.17819416D-02        UMP4(DQ)=   -0.38305431602D+02
 E4(SDQ)=  -0.20515822D-02        UMP4(SDQ)=  -0.38305701243D+02
 DE(Corr)= -0.99901549E-01 E(Corr)=     -38.301729156    
 NORM(A)=   0.10198723D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 DE(Corr)= -0.10324771     E(CORR)=     -38.305075315     Delta=-3.35D-03
 NORM(A)=   0.10215365D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 DE(Corr)= -0.10406744     E(CORR)=     -38.305895046     Delta=-8.20D-04
 NORM(A)=   0.10218097D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 DE(Corr)= -0.10419872     E(CORR)=     -38.306026331     Delta=-1.31D-04
 NORM(A)=   0.10218422D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 DE(Corr)= -0.10420639     E(CORR)=     -38.306034000     Delta=-7.67D-06
 NORM(A)=   0.10218410D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 DE(Corr)= -0.10420612     E(CORR)=     -38.306033723     Delta= 2.76D-07
 NORM(A)=   0.10218413D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 DE(Corr)= -0.10420633     E(CORR)=     -38.306033937     Delta=-2.14D-07
 NORM(A)=   0.10218413D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        14
 NAB=     4 NAA=     6 NBB=     0.
 DE(Corr)= -0.10420638     E(CORR)=     -38.306033984     Delta=-4.68D-08
 NORM(A)=   0.10218412D+01
 Largest amplitude= 4.49D-02
 Time for triples=        1.77 seconds.
 T4(CCSD)= -0.25711839D-02
 T5(CCSD)=  0.28947080D-04
 CCSD(T)= -0.38308576221D+02

 S**2, projected HF & approx projected MPn energies after annihilation of
 unwanted spin states (see manual for definitions):

 spins       (S**2,0) (S**2,1)    PUHF        PMP2        PMP3        PMP4
 annihilated
 s+1         3.75000  3.75000  -38.202552  -38.289955  -38.303815

 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (?A) (?A) (?B) (?B) (?C)
       Virtual   (?A) (?B) (?B) (?B) (?B) (?A) (?C) (?D) (?D) (?B)
                 (?B) (?A) (?A) (?A) (?B) (?B) (?C) (?D) (?D) (?B)
                 (?B) (?B) (?B) (?A) (?C) (?A) (?B) (?B) (?B) (?B)
                 (?D) (?D) (?A) (?B) (?B) (?C) (?A) (?D) (?D) (?C)
                 (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?A) (?C)
                 (?B) (?B) (?D) (?D) (?C) (?A) (?B) (?B) (?B) (?B)
                 (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Beta  Orbitals:
       Occupied  (?A) (?A)
       Virtual   (?B) (?B) (?C) (?A) (?B) (?B) (?B) (?B) (?A) (?C)
                 (?A) (?B) (?B) (?D) (?D) (?A) (?A) (?B) (?B) (?C)
                 (?D) (?D) (?B) (?B) (?B) (?B) (?A) (?A) (?C) (?B)
                 (?B) (?B) (?B) (?D) (?D) (?A) (?C) (?B) (?B) (?A)
                 (?D) (?D) (?C) (?A) (?B) (?B) (?D) (?D) (?B) (?B)
                 (?A) (?A) (?C) (?B) (?B) (?D) (?D) (?C) (?B) (?B)
                 (?A) (?B) (?B) (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Unable to determine electronic state:  an orbital has unidentified symmetry.
 Alpha  occ. eigenvalues --  -12.12440  -1.85239  -1.48592  -1.48592  -1.21867
 Alpha virt. eigenvalues --   -0.39466  -0.34138  -0.34138  -0.21925  -0.21925
 Alpha virt. eigenvalues --   -0.17545  -0.15266   0.01895   0.01895   0.01911
 Alpha virt. eigenvalues --    0.01911   0.04363   0.34008   0.38675   0.39962
 Alpha virt. eigenvalues --    0.39962   0.49275   0.60111   0.60111   0.77471
 Alpha virt. eigenvalues --    0.77471   0.85622   0.85622   1.06129   1.56605
 Alpha virt. eigenvalues --    1.58236   1.74368   1.74368   2.03490   2.03490
 Alpha virt. eigenvalues --    2.05167   2.05167   2.29874   2.38072   2.38072
 Alpha virt. eigenvalues --    2.38145   2.39054   2.46243   2.46243   2.69714
 Alpha virt. eigenvalues --    2.73119   2.87177   2.87177   3.10392   3.10392
 Alpha virt. eigenvalues --    3.14539   3.14539   3.38917   3.40941   3.41376
 Alpha virt. eigenvalues --    3.45409   3.45409   3.60593   3.60593   4.18244
 Alpha virt. eigenvalues --    4.32239   4.34825   4.34825   4.40137   4.40137
 Alpha virt. eigenvalues --    4.46266   4.46266   5.05034   5.14669   5.14669
 Alpha virt. eigenvalues --    5.68486  12.42120
  Beta  occ. eigenvalues --  -12.06972  -1.47416
  Beta virt. eigenvalues --   -0.82499  -0.82499  -0.55940  -0.33736  -0.26136
  Beta virt. eigenvalues --   -0.26136  -0.16655  -0.16655  -0.07314  -0.05901
  Beta virt. eigenvalues --    0.08369   0.09354   0.09354   0.11965   0.11965
  Beta virt. eigenvalues --    0.37846   0.44985   0.46174   0.46174   0.53675
  Beta virt. eigenvalues --    0.63932   0.63932   0.80718   0.80718   0.91016
  Beta virt. eigenvalues --    0.91016   1.13862   1.66276   1.69768   1.82017
  Beta virt. eigenvalues --    1.82017   2.11025   2.11025   2.16698   2.16698
  Beta virt. eigenvalues --    2.38202   2.46584   2.47324   2.47324   2.51468
  Beta virt. eigenvalues --    2.54617   2.54617   2.73185   2.77657   2.92012
  Beta virt. eigenvalues --    2.92012   3.14752   3.14752   3.19287   3.19287
  Beta virt. eigenvalues --    3.41271   3.44187   3.45246   3.49591   3.49591
  Beta virt. eigenvalues --    3.63810   3.63810   4.22210   4.38611   4.38611
  Beta virt. eigenvalues --    4.38649   4.43818   4.43818   4.51658   4.51658
  Beta virt. eigenvalues --    5.10384   5.20770   5.20770   5.72529  12.45839
          Condensed to atoms (all electrons):
               1          2          3          4
     1  C    4.853854   0.260957   0.260957   0.260957
     2  H    0.260957   0.190368   0.001550   0.001550
     3  H    0.260957   0.001550   0.190368   0.001550
     4  H    0.260957   0.001550   0.001550   0.190368
          Atomic-Atomic Spin Densities.
               1          2          3          4
     1  C    1.716901   0.147469   0.147469   0.147469
     2  H    0.147469   0.137306  -0.002272  -0.002272
     3  H    0.147469  -0.002272   0.137306  -0.002272
     4  H    0.147469  -0.002272  -0.002272   0.137306
 Mulliken charges and spin densities:
               1          2
     1  C    0.363274   2.159307
     2  H    0.545575   0.280231
     3  H    0.545575   0.280231
     4  H    0.545575   0.280231
 Sum of Mulliken charges =   2.00000   3.00000
 Mulliken charges and spin densities with hydrogens summed into heavy atoms:
               1          2
     1  C    2.000000   3.000000
 Electronic spatial extent (au):  <R**2>=             15.8310
 Charge=              2.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              0.4394   YY=              0.4394   ZZ=             -5.2789
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              1.9061   YY=              1.9061   ZZ=             -3.8122
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              3.0517  ZZZ=              0.0000  XYY=              0.0000
  XXY=             -3.0517  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -0.4476 YYYY=             -0.4476 ZZZZ=             -4.1324 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -0.1492 XXZZ=             -1.7008 YYZZ=             -1.7008
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 9.643716840664D+00 E-N=-1.029471156840D+02  KE= 3.896719354587D+01
 Symmetry A1   KE= 3.623604249645D+01
 Symmetry A2   KE=-2.093046613144D-19
 Symmetry B1   KE= 1.225925677202D+00
 Symmetry B2   KE= 1.505225372217D+00
 1\1\GINC-GAMBA\SP\UCCSD(T)-FC\Gen\C1H3(2+,4)\IRIKURA\17-May-2017\0\\# 
 CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc guess=check\\BEB ste
 p 9: high-spin CCSD(T) for dication of ch3\\2,4\C,0,0.,-0.0000000011,0
 .\H,0,0.000000002,-1.0827517802,0.\H,0,-0.9376905476,0.5413758867,0.\H
 ,0,0.9376905457,0.5413758901,0.\\Version=ES64L-G09RevD.01\HF=-38.20182
 76\MP2=-38.2895722\MP3=-38.3036497\MP4D=-38.307354\MP4DQ=-38.3054316\P
 UHF=-38.2025518\PMP2-0=-38.2899552\PMP3-0=-38.3038154\MP4SDQ=-38.30570
 12\CCSD=-38.306034\CCSD(T)=-38.3085762\S2=3.752981\S2-1=3.750172\S2A=3
 .75\RMSD=2.717e-09\PG=D03H [O(C1),3C2(H1)]\\@


 Cherishing children is the mark of a civilized society.
 -- Joan Ganz Cooney
 Job cpu time:       0 days  0 hours  0 minutes 25.6 seconds.
 File lengths (MBytes):  RWF=     41 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:00:24 2017.
